deploy/ui/form.json

```diff
@@ -47,6 +47,14 @@
             }
           ]
         },
+        {
+          "label": "Disable Disconnected Mode",
```

Comment:
Shouldn't we just say "Use Image Tags" here and explain the consequences in the description as you do?

Reply:
done